modules/apps/headless/headless-form/headless-form-api/src/main/java/com/liferay/headless/form/dto/v1_0/util/StructureUtil.java

```diff
@@ -192,6 +194,7 @@ private static FormField _toFormField(
 				repeatable = ddmFormField.isRepeatable();
 				required = ddmFormField.isRequired();
 				showLabel = ddmFormField.isShowLabel();
+				placeholder = _toString(locale, placeholderLocalizedValue);
```

Comment:
@joaovictoralvesmartins, can you please add this line before `predefinedValue` variable to keep alphabetical sort?

Thanks.

Reply:
Done!